.github/workflows/Pipeline.yml

```diff
@@ -1,274 +1,106 @@
-
-      - name: ⚙ Install dependencies for packaging and release
-        run: |
-          python -m pip install --upgrade pip
-          pip install wheel twine
-
-      - name: ⤴ Release Python package to PyPI
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-        run: |
-          twine upload dist/*
+    with:
+      python_version: ${{ fromJson(needs.Params.outputs.params).python_version }}
+      artifact: ${{ fromJson(needs.Params.outputs.params).artifacts.package }}
+    secrets:
+      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
 
   BuildTheDocs:
-    name: 📓 Run BuildTheDocs and publish to GH-Pages
-    runs-on: ubuntu-latest
-
-#    needs:
-#      - VerifyDocs
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: 🚢 Build container image 'vhdl/doc'
-        run: |
-          docker build -t vhdl/doc - <<-EOF
-          FROM btdi/sphinx:featured
-          RUN apk add -U --no-cache graphviz
-          EOF
-
-      - name: 🛳️ Build documentation from './pyEDAA.OutputFilter/doc'
-        uses: buildthedocs/btd@v0
-        with:
-          token: ${{ github.token }}
+    uses: pyTooling/Actions/.github/workflows/BuildTheDocs.yml@r0
+    needs:
+      - Params
+    with:
+      artifact: ${{ fromJson(needs.Params.outputs.params).artifacts.doc }}
 
-      - name: 📤 Upload artifacts to GitHub Pages
-        uses: actions/upload-artifact@master
-        with:
-          name: doc
-          path: doc/_build/html
+  PublishToGitHubPages:
+    uses: pyTooling/Actions/.github/workflows/PublishToGitHubPages.yml@r0
+    needs:
+      - Params
+      - BuildTheDocs
+      #- Coverage
+    with:
+      doc: ${{ fromJson(needs.Params.outputs.params).artifacts.doc }}
+      #coverage: ${{ fromJson(needs.Params.outputs.params).artifacts.coverage }}
 
   ArtifactCleanUp:
-    name: 🗑️ Artifact Cleanup
-    runs-on: ubuntu-latest
-
+    uses: pyTooling/Actions/.github/workflows/ArtifactCleanUp.yml@r0
     needs:
-      - Package
-      - PublishOnPyPI
-
-    env:
-      ARTIFACT: ${{ needs.Package.outputs.artifact }}
-
-    steps:
-      - name: 🗑️ Delete all Artifacts
-        uses: geekyeggo/delete-artifact@v1
-        with:
-          name: |
-            ${{ env.ARTIFACT }}
+      - Params
+      #- UnitTesting
+      #- Coverage
+      - BuildTheDocs
+      - PublishToGitHubPages
+    with:
+      package: ${{ fromJson(needs.Params.outputs.params).artifacts.package }}
+      remaining: |
+        ${{ fromJson(needs.Params.outputs.params).artifacts.doc }}
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.6
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.7
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.8
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.9
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.coverage }}
```

Comment:
Missing line break according to GH online diff view.

Reply:
Done.